integration-test/tests/login-form.ios.spec.js

```diff
@@ -86,6 +86,27 @@ test.describe('Auto-fill a login form on iOS', () => {
                 await login.promptWasNotShown()
             })
         })
+
+        test.describe('check tooltip opening logic', () => {
+            test('tapping into an autofilled field does not prompt', async ({page}) => {
+                const {login} = await testLoginPage(page, server, {
+                    featureToggles: {
+                        inputType_credentials: true
+                    },
+                    availableInputTypes: {
+                        credentials: true
+                    },
+                    credentials
+                })
+                await login.promptWasShown('ios')
+                await page.pause()
+
+                await login.clickIntoPasswordInput()
+                const calls = await page.evaluate('window.__playwright.mocks.calls')
+                const mockCalls = calls.filter(([name]) => name === 'getAutofillData')
+                expect(mockCalls).toHaveLength(1)
```

Comment:
Can we move these 3 lines into a method in pages? something like 

```javascript
await login.assertAutofillDataRequested()
```

I also think there's a helper for mock calls, something like: 

```javascript
const calls = await mockCalls(page, ["getAutofillData"]);
expect(calls).toHaveLength(1)
```

Reply:
The helper uses `find`, here we must check the exact number of calls (1 vs 2).